### DISCUSSION TITLE
How to modify chart template

### DISCUSSION THREAD

**Discussion by bzmatteo72** [apache/superset#13337] [Q&A / Help]
hello,
I'm new to Superset, please I would like to make a change to the standard template for chart visualization type, for example I need to remove the (default) scroll bar from the "Big Number" chart type.
Please could you indicate to me if there is any way to customize the existing chart visualization type and create a new one?
Thanks

Matteo

**Comment by kamalkeshavani-aiinside** [comment]
Check this:
https://superset.apache.org/docs/frequently-asked-questions#how-do-i-create-my-own-visualization

**Reply by bzmatteo72** [reply]
hi,
thanks for your response. 
I actually had already read this documentation but it doesn't explain where are saved in the file system (on the machine where Superset is installed I mean) the  scripts/packages that contain already existing chart templates for the different visualization types.
In this document as far as I have understood it is just explaining how to create new template from scratch.
Instead i need to modify/customize the existing chart visualization type, not creating new ones.
Do you know if there is anyboby in the Superset community who can provide help with this?
Thanks anyway
Matteo

**Reply by francis-barre** [reply]
Hi @bzmatteo72, have you found a solution since? I am trying to modify one of the template charts but couldn't find their location either. 

Thanks 
Francis

**Reply by OrwaRebootMob** [reply]
`./superset-frontend/plugins/plugins-chart-echarts`. In the `README.md` in that folder there is a mention of a storybook interface served [here](https://apache-superset.github.io/superset-ui/?path=/story/chart-plugins-plugin-chart-echarts-pie--weekday-pie)
